source/air/src/ast.rs

```diff
@@ -156,16 +144,16 @@ pub enum ExprX {
     Bind(Bind, Expr),
     // Sometimes an axiom will have additional error messages. If an assert fails
     // and this axiom was relevant, then we append the error labels to the Message.
-    LabeledAxiom(MessageLabels, Expr),
-    LabeledAssertion(Message, Expr),
+    LabeledAxiom(Vec<Arc<dyn Any + Send + Sync>>, Expr),
```

Comment:
Can we make a type abbreviation or a struct wrapper for `dyn Any + Send + Sync`?